nengo_interfaces/tests/test_changing_airsim_settings.py

```diff
@@ -19,16 +24,19 @@
     f"{home}/Documents/AirSim/settings.json", "r+"
 ) as fp:
     data = json.load(fp)
-    prev_width = data["CameraDefaults"]["CaptureSettings"][0]["Width"]
+    try:
+        prev_width = data["CameraDefaults"]["CaptureSettings"][0]["Width"]
+    except KeyError:
+        print(
+            f"{yellow}No CameraDefaults found in settings.json, using default airsim values for testing{endc}"
+        )
+        prev_width = 256
```

Comment:
This test still fails for me when I use a `settings.json` without `CameraDefaults`. Line 257 in `airsim.py` assumes it is there and gives a `KeyError`

This is the settings file I tested it with:
```
{
  "SeeDocsAt": "https://github.com/Microsoft/AirSim/blob/master/docs/settings.md",
  "SettingsVersion": 1.2,
  "SimMode": "Multirotor",
  "ViewMode": "SpringArmChase"
}
```

Reply:
Nice catch, fixed now. I updated it to check if the key exists, if not it fills it with the default in cv_settings.json.